lib/AST/Module.cpp

```diff
@@ -1472,6 +1472,15 @@ void SourceFile::setTypeRefinementContext(TypeRefinementContext *Root) {
   TRC = Root;
 }
 
+void SourceFile::createReferencedNameTracker(
+    StringRef referenceDependenciesFilePath) {
+  if (!referenceDependenciesFilePath.empty()) {
```

Comment:
Ah, if this is conditional, then the method should probably have an "ifNeeded" in the name.

Reply:
Good catch--fixed!